metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -97,6 +101,61 @@ class GEProfilerRequest:
     batch_kwargs: dict
 
 
+def get_column_unique_count_patch(self, column):
+    if self.engine.dialect.name.lower() == "redshift":
+        element_values = self.engine.execute(
+            f"SELECT APPROXIMATE count (distinct {column}) FROM {self._table}"
```

Comment:
I'd like to lean on SQLAlchemy where possible. Would something along these lines work (might need a `sa.column` in there as well)?

```suggestion
            sa.select([f"APPROXIMATE count (distinct {column})"]).select_from(self._table)
```